schema/m3-room.schema.json

```diff
@@ -0,0 +1,1985 @@
+          "additionalProperties": false,
+          "minProperties": 1,
+          "maxProperties": 1,
+          "properties": {
+            "comeInRunning": {
+              "$id": "#/definitions/strat/properties/entranceCondition/properties/comeInRunning",
+              "type": "object",
+              "title": "Come In Running",
+              "description": "Represents a requirement to run in through this door with speed in a specified range.",
+              "required": [
+                "speedBooster",
+                "minTiles"
+              ],
+              "additionalProperties": false,
+              "properties": {
+                "speedBooster": {
+                  "$id": "#/definitions/strat/properties/entranceCondition/properties/comeInRunning/properties/speedBooster",
+                  "type": ["boolean", "string"],
+                  "title": "Speed Booster",
+                  "description": "Whether or not this run should be performed with Speed Booster.",
+                  "enum": [
+                    true,
+                    false,
+                    "any"
+                  ]
+                },
+                "minTiles": {
+                  "$id": "#/definitions/strat/properties/entranceCondition/properties/comeInRunning/properties/minTiles",
+                  "type": "number",
+                  "title": "Minimum Speed in Tiles",
+                  "description": "The minimum horizontal speed that will satisfy this cross-room requirement, measured in effective runway tiles with dash held."
+                },
+                "maxTiles": {
+                  "$id": "#/definitions/strat/properties/entranceCondition/properties/comeInRunning/properties/maxTiles",
+                  "type": "number",
+                  "title": "Maximum Speed in Tiles",
+                  "description": "The maximum horizontal speed that will satisfy this cross-room requirement, measured in effective runway tiles with dash held."
+                }
+              }
+            },
+            "comeInJumping": {
+              "$id": "#/definitions/strat/properties/entranceCondition/properties/comeInJumping",
+              "type": "object",
+              "title": "Come In Jumping",
+              "description": "Represents a requirement that Samus come in by jumping through the door transition.",
+              "required": [
+                "speedBooster",
+                "minTiles"
+              ],
+              "additionalProperties": false,
+              "properties": {
+                "speedBooster": {
+                  "$id": "#/definitions/strat/properties/entranceCondition/properties/comeInJumping/properties/speedBooster",
+                  "type": ["boolean", "string"],
+                  "title": "Speed Booster",
+                  "description": "Whether or not this run should be performed with Speed Booster.",
+                  "enum": [
+                    "any"
+                  ]
```

Comment:
This enum is different from comeInRunning

Reply:
Yeah, it needs to be fixed. Maybe it can be done in a separate PR, to keep the history cleaner, since there's so much going on in this one. @kjbranch already found it needed to be fixed in #1128 so maybe it can be adapted there after this is merged